assets/src/components/carousel/index.js

```diff
@@ -124,7 +125,7 @@ export function Carousel( {
 					) ) }
 				</ul>
 			</div>
-			{ currentPage && (
+			{ showCarouselNav && (
```

Comment:
I noticed the carousel navigation is not updated correctly when viewed on mobile:


https://user-images.githubusercontent.com/16200219/107400384-aace9b80-6af9-11eb-8698-8388a3c63977.mp4


A workaround may be to wrap `showCarouselNav` in a `useEffect` hook:

```diff
diff --git a/assets/src/components/carousel/index.js b/assets/src/components/carousel/index.js
--- a/assets/src/components/carousel/index.js	(revision f14aa035c6830468fb1059924477ce6688300a4d)
+++ b/assets/src/components/carousel/index.js	(date 1612887747254)
@@ -38,6 +38,7 @@
 	const { windowWidth } = useWindowWidth();
 	const [ currentPage, originalSetCurrentPage ] = useState( null );
 	const [ pageWidth, setPageWidth ] = useState( 0 );
+	const [ showCarouselNav, setShowCarouselNav ] = useState( null );
 	const carouselListRef = useRef();
 
 	/**
@@ -103,7 +104,13 @@
 		setPageWidth( carouselListRef?.current?.clientWidth || 0 );
 	}, [ items.length, windowWidth ] );
 
-	const showCarouselNav = currentPage && items.length > 1;
+	/**
+	 * Show the carousel navigation when there is more than one page.
+	 */
+	useEffect( () => {
+		setShowCarouselNav( items.length > 1 );
+	}, [ items.length ] );
+
 	const centeredItemIndex = [ ...( carouselListRef.current?.children || [] ) ].indexOf( currentPage );
 	const nextButtonDisabled = centeredItemIndex >= items.length - 1;
 	const prevButtonDisabled = centeredItemIndex <= 0;
```

Reply:
Thank you, @pierlon! I've addressed it in 5934dd8.